pipeline/titanic/modeling_code.py

```diff
@@ -0,0 +1,94 @@
+from pyspark.ml import Pipeline
+from pyspark.sql.functions import (
+    mean, col, split,
+    regexp_extract, when, lit)
+
+from pyspark.ml.feature import (
+    VectorAssembler,
+    StringIndexer
+)
+
+TRAINING_DF_INDEX = 0
+TESTING_DF_INDEX = 1
+
+training_df = training_df.withColumnRenamed('Survived', 'label')
+testing_df = testing_df.withColumn('label', lit(0))
+datasets_list = [training_df, testing_df]
+
+for index, dataset in enumerate(datasets_list):
+    dataset = dataset.withColumn(
+        "Initial",
+        regexp_extract(col("Name"), "([A-Za-z]+)\.", 1))
+    datasets_list[index] = dataset
+
+misspelled_initials = [
+    'Mlle', 'Mme', 'Ms', 'Dr',
+    'Major', 'Lady', 'Countess',
+    'Jonkheer', 'Col', 'Rev',
+    'Capt', 'Sir', 'Don'
+]
+correct_initials = [
+    'Miss', 'Miss', 'Miss', 'Mr',
+    'Mr', 'Mrs', 'Mrs',
+    'Other', 'Other', 'Other',
+    'Mr', 'Mr', 'Mr'
+]
+for index, dataset in enumerate(datasets_list):
+    dataset = dataset.replace(misspelled_initials, correct_initials)
+    datasets_list[index] = dataset
+
+initials_age = {"Miss": 22,
```

Comment:
Can you also change the constants variables that has in this code to caps lock? e.g. correct_initials to CORRECT_INITIALS,